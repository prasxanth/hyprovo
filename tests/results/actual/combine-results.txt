2021-11-24 14:41:21.234 - INFO - Pass ✓...  add pass
2021-11-24 14:41:21.237 - ERROR - Fail ✗...  sub fail
2021-11-24 14:41:21.240 - INFO - Pass ✓...  mul pass
2021-11-24 14:41:21.246 - INFO - Pass ✓...  div pass
2021-11-24 14:41:21.249 - ERROR - Fail ✗...  add fail
2021-11-24 14:41:21.252 - ERROR - Fail ✗...  mul fail
