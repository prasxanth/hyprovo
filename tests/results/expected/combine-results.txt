2021-11-24 14:10:30.733 - INFO - Pass ✓...  add pass
2021-11-24 14:10:30.736 - ERROR - Fail ✗...  sub fail
2021-11-24 14:10:30.738 - INFO - Pass ✓...  mul pass
2021-11-24 14:10:30.743 - INFO - Pass ✓...  div pass
2021-11-24 14:10:30.745 - ERROR - Fail ✗...  add fail
2021-11-24 14:10:30.747 - ERROR - Fail ✗...  mul fail
